ui/app/css/itcss/components/network.scss

```diff
@@ -43,3 +43,66 @@
 .network-check__transparent {
   opacity: 0;
 }
+
+.menu-icon-circle, .menu-icon-circle--active {
+  height: 23px;
+  width: 23px;
+  margin: 9px;
+  background: none;
+  border-radius: 22px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: none;
```

Comment:
instead of border none, we should do `1px solid transparent`. This would keep the circle the same size with or without border.

Alternatively, we can set `box-sizing` to `border-box`

Reply:
done in ca5e276e